pkg/scheduler/apis/config/types.go

```diff
@@ -88,6 +96,14 @@ type KubeSchedulerConfiguration struct {
 	// If this value is nil, the default value will be used.
 	BindTimeoutSeconds *int64
 
+	// PodInitialBackoffDurationSeconds is the initial backoff duration for unschedulable pods.
+	// It should be greater than 0.
+	PodInitialBackoffDurationSeconds int64 `json:"podInitialBackoffDurationSeconds"`
```

Comment:
shouldn't this be a pointer like `BindTimeoutSeconds`?

Reply:
SG